src/components/FileUploader/FileUploader-story.js

```diff
@@ -36,7 +39,7 @@ const props = {
       labelText: text('Label text (labelText)', 'Add files'),
       name: text('Form item name: (name)', ''),
       multiple: boolean('Supports multiple files (multiple)', true),
-      buttonKind: buttonKind === 'none' ? '' : buttonKind,
+      buttonKind: buttonKind || 'primary',
```

Comment:
Hi @emyarod any reason behind this change…? Thanks!

Reply:
hi @asudoh I wanted to consult you about this change but this is what I saw: I removed the 'none' option entirely since the value of `buttonKind` was defaulting to empty string [here](https://github.com/emyarod/carbon-components-react/blob/09b98f8fabcbb612c25dce086ef6cc56f5bbb230/src/components/FileUploader/FileUploader-story.js#L36) I believe. So it looked like the value of `buttonKind` was never `'none'`, and I changed it to default to `'primary'` if the value of `buttonKind` was falsy

Reply:
Thank you for the background @emyarod! My bad that I put an empty string there, which should be `none` aligned to the 1st selection my original code. I made it simply because empty string as an object key it not natural (though the spec seems to say it’s valid). Great to keep the selection of empty `buttonKind` in either way - Thanks!

Reply:
would you like me to modify it to be `'none'` instead of empty string, in line with your original intention @asudoh ? maybe `null` would work better than empty string too, since that it be just like if the user did not pass in any value for the `buttonKind` prop

Reply:
Thank you for asking @emyarod! Let’s go with `none` for now, given `null` won’t work as a object key (which has to be a string).

Reply:
oh ok so just to clarify @asudoh, is [this](https://github.com/IBM/carbon-components-react/pull/1202) more in line with what you were looking for?